src/Builder/FileNormalizer/PhpCSFixerFileNormalizer.php

```diff
@@ -47,12 +52,15 @@ public function __construct(IntegratorConfig $config)
      */
     public function normalize(array $filePaths): void
     {
-        $process = new Process([$this->getCSFixPath(), ...$this->getAbsoluteFilePaths($filePaths)]);
-        $process->setTimeout(static::PROCESS_TIMEOUT);
-        $process->run();
+        $projectConsolePath = $this->getProjectConsolePath();
+        foreach ($this->getProjectRelativeFilePaths($filePaths) as $filePath) {
```

Comment:
We need this loop because the console contains only `subPathKey (string)`  (array is not possible) https://github.com/spryker/development/blob/master/src/Spryker/Zed/Development/Communication/Console/CodeStyleSnifferConsole.php#L56

Reply:
I dont get this

> $this->addArgument(static::ARGUMENT_SUB_PATH, InputArgument::OPTIONAL, 'Optional path or sub path element for project level');

is an optional piece that would never be used for integrator run IMO.

Reply:
Yes, it can be skipped. We try to use the option in PhpCS command to use the normalizer only for files that were changed by integrator